util/jsonutil/keywordsutil.go

```diff
@@ -0,0 +1,59 @@
+package jsonutil
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+)
+
+type KeywordsString string
+
+// extImpAppnexusKeyVal defines the contract for bidrequest.imp[i].ext.prebid.bidder.appnexus.keywords[i]
+type extImpAppnexusKeyVal struct {
+	Key    string   `json:"key,omitempty"`
+	Values []string `json:"value,omitempty"`
+}
```

Comment:
This is not a shareable type since its specifically for the AppNexus data model. It is specific to the AppNexus adapter. I recommend we move this type to `imp_appnexus.go`.

Reply:
Moving this to `imp_appnexus.go` creates a cyclic dependency. As part of [45e4be9](https://github.com/prebid/prebid-server/pull/2649/commits/45e4be90109764dd51d69ab096c61c0ebfaab107), I have renamed this struct to `KeyVals`